libs/hdf-converters/src/veracode-mapper.ts

```diff
@@ -0,0 +1,587 @@
+    for (let index = 0; index < len; index++) {
+      const cwe = cweloop(input, index);
+      text.push(cwe);
+    }
+  }
+  return text.join('\n');
+}
+
+function formatCweDesc(input: Record<string, unknown>): string {
+  const text = [];
+  if (_.has(input, 'cwe')) {
+    const len = Number(`${_.get(input, CWE_LENGTH)}`);
+    for (let index = 0; index < len; index++) {
+      const cweid = `cwe[${index}].cweid`;
+      let cwe = 'CWE-'.concat(`${_.get(input, cweid)}`) + ': ';
+      const cwename = `cwe[${index}].cwename`;
+      const desc = `cwe[${index}].description.text.text`;
+      cwe += `${_.get(input, cwename)}` + ': ';
+      cwe += `${_.get(input, desc)}`;
+      text.push(cwe);
+    }
+  }
+  return text.join('\n');
+}
+
+function formatSourceLocation(input: Record<string, unknown>): string {
+  const text = [];
+  const len = Number(`${_.get(input, 'length')}`);
+  for (let index = 0; index < len; index++) {
+    const source = `[${index}].sourcefile`;
+    text.push(`${_.get(input, source)}`);
+  }
+  return text.join('\n');
+}
+
+function stringifyCodeDesc(input: Record<string, unknown>): string {
+  const categories = [
+    ['Line Number', 'line'],
+    ['Affect Policy Compliance', 'affects_policy_compliance'],
+    ['Remediation Effort', 'remediationeffort'],
+    ['Exploit level', 'exploitLevel'],
+    ['Issue ID', 'issueid'],
+    ['Module', 'module'],
+    ['Type', 'type'],
+    ['CWE ID', 'cweid'],
+    ['Date First Occurence', 'date_first_occurence'],
+    ['CIA Impact', 'cia_impact'],
+    ['Description', 'description']
+  ];
+  let flawDesc =
+    'Sourcefile Path: ' + `${_.get(input, 'sourcefilepath')}` + ';';
+  categories.forEach(function (value) {
+    if (_.has(input, value[1])) {
+      flawDesc += `${value[0]}: ` + `${_.get(input, value[1])}` + ';';
+    }
+  });
+  return flawDesc;
+}
+
+function formatCodeDesc(input: Record<string, unknown>): string {
+  const text = [];
+  if (`${_.get(input, 'sourcefilepath')}` !== 'undefined') {
+    const flawDesc = stringifyCodeDesc(input);
+    text.push(flawDesc);
+  }
+  return text.join('\n');
+}
+
+function SCAstringify(input: Record<string, unknown>): string {
+  const categories = [
+    'sha1',
+    'max_cvss_score',
+    'version',
+    'library',
+    'vendor',
+    'descritpion',
+    'added_date',
+    'component_affects_policy_compliance'
+  ];
+  let flawDesc = 'component_id: ' + `${_.get(input, 'component_id')}` + ';';
+  categories.forEach(function (value) {
+    if (_.has(input, value)) {
+      flawDesc += `${value}: ` + `${_.get(input, value)}` + ';';
+    }
+  });
+  if (`${_.has(input, 'file_paths.file_path.value')}`) {
+    flawDesc +=
+      'file_path: ' + `${_.get(input, 'file_paths.file_path.value')}` + ';';
+  }
+  return flawDesc;
+}
+
+function formatSCACodeDesc(input: Record<string, unknown>): string {
+  const text = [];
+  if (`${_.get(input, 'component_id')}` !== 'undefined') {
+    const flawDesc = SCAstringify(input);
```

Comment:
in most cases, separating out these stringify functions is unnecessary unless it'd be drastically clearer being split out or if you're running into function length caps as set by the linter.  so i would again recommend inlining that function into here.

also like above there's no iteration going on here but you're still doing the array thing.  either the iteration is missing or the array stuff is unnecessary.